### PR TITLE
Add code lens for "Open referenced file"

### PR DESCRIPTION
**Pull request by shati-patel**
Makes the "Open referenced file" command more discoverable by adding a code lens: 
image

Click that prompt to jump to the referenced QL file 😎 See internal issue for more details! 🔍 

### Notes:
- I put this in the `local-queries` folder, which seemed reasonable 😅 Happy to be pointed to a more suitable location!
- I couldn't think of a useful way to test this, since it just calls out to the existing `codeQL.openReferencedFile` command... 

## Checklist

- [x] [CHANGELOG.md](https://github.com/github/vscode-codeql/blob/main/extensions/ql-vscode/CHANGELOG.md) has been updated to incorporate all user visible changes made by this pull request.
- [x] Issues have been created for any UI or other user-facing changes made by this pull request.
- [x] _[Maintainers only]_ If this pull request makes user-facing changes that require documentation changes, open a corresponding docs pull request in the [github/codeql](https://github.com/github/codeql/tree/main/docs/codeql/codeql-for-visual-studio-code) repo and add the `ready-for-doc-review` label there.

### CODE REVIEW

**Review by robertbrignull — APPROVED** [review]
You may want to wait for a review from @github/codeql-vscode-reviewers, but the code looks good to me and I tried testing it and the codelens worked and was intuitive.

**Review by aeisenberg — COMMENTED** [review]
That's a great idea. Thanks for implementing!

**Pull request merged**
